src/main/java/bio/terra/cli/command/shared/options/WorkspaceNameAndDescription.java

```diff
@@ -2,7 +2,9 @@
 
 import picocli.CommandLine;
 
-/** @CommandLine.Mixin class for workspace name and description options */
```

Comment:
I believe we're following Google Java style guide. Having it on one line is valid...

https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment:~:text=...%20or%20in%20this%20single%2Dline%20example%3A

Reply:
This was change was auto-included when formatting was applied, will recheck